Project-popup.js

```diff
@@ -0,0 +1,198 @@
+const PopupArray = [{
+  title: 'Multi-Post Stories',
+  titleDesktop: 'Keeping track of hundreds  of components website',
+  img: './images/popup-background-image.png',
+  text1: 'Lorem Ipsum is simply dummy text of the <br> printing and typesetting industry. Lorem <br> Ipsum has been the industrys standard <br> dummy text ever since the 1500s, when <br> an unknown printer took a galley of type <br> and scrambled it to make a type <br> specimen book. It has survived not only <br> five centuries, but also the leap into <br> electronic typesetting, remaining essent',
+  text2: 'Lorem Ipsum is simply dummy text of the printing and <br> typesetting industry. Lorem Ipsum has been the <br> industrys standard dummy text ever since the 1500s, <br> when an unknown printer took a galley of type and <br> scrambled it 1960s with the releaLorem Ipsum is simply <br> dummy text of the printing and typesetting industry. <br> Lorem Ipsum has been the industrys standard dummy <br> text ever since the 1500s, when an unknown printer <br> took a galley of type and scrambled it 1960s with the <br> releax map lapora verita.',
+  techstack1: 'html',
+  techstack2: 'Bootstrap',
+  techstack3: 'Ruby on rails',
+  button1: 'See Live',
+  button2: 'See Source',
+  button1Img: './images/popup-icon-see project.png',
+  button2Img: './images/popup-icon-github.png',
+}];
+
```

Comment:
- [ ] When you click on see project kindly populate the popup with the content of the selected project